test/test.js

```diff
@@ -57,9 +57,22 @@ test('fly-rollup', t => {
         t.true(res.indexOf('# sourceMappingURL=data:application/json;base64,{"version":3,') > -1, 'appends `sourceMappingURL` content');
```

Comment:
The unnecessary `{"version":3,` here is failing the test, not the [`new Buffer(...)` line](https://github.com/frantic1048/fly-rollup/pull/4/files#diff-168726dbe96b3ce427e7fedce31bb0bcR29). However the brackets make it more readable, let's keep them.

The JSON object output by rollup actually containing `"version":3`, but it is base64 encoded after `# sourceMappingURL=data:application/json;base64,` . Then the `.indexOf()` fails with `{"version":3,`.

We could remove `{"version":3,` here.

Could you help fix here and squash the commits so that I can merge your PR ?

Thanks for your kind work :)